corehq/apps/export/export.py

```diff
@@ -402,37 +395,10 @@ def _time_in_milliseconds():
     return int(time.time() * 1000)
 
 
-def _record_datadog_export_compute_rows(duration, doc_bytes, n_rows, tags):
-    __record_datadog_export(duration, doc_bytes, n_rows, 'commcare.compute_export_rows_duration', tags)
-
-
-def _record_datadog_export_write_rows(duration, doc_bytes, n_rows, tags):
-    __record_datadog_export(duration, doc_bytes, n_rows, 'commcare.write_export_rows_duration', tags)
```

Comment:
removed since I don't think it's used. It was added when digging into export performance